docs/src/tutorial_lightgraphs.md

```diff
@@ -0,0 +1,107 @@
+# LightGraphs.jl  interface
+
+```jldoctest lightgraphs
+julia> using LightGraphs
+
+julia> using MetaGraphsNext
+```
+
+`MetaGraph`s inherit many methods from LightGraphs.jl. In general, inherited methods refer to vertices by codes, not labels, for compatibility with AbstractGraph. Vertex codes get reassigned after `rem_vertex!` to remain contiguous, so we recommend using labels if possible.
```

Comment:
```suggestion
`MetaGraph`s inherit many methods from LightGraphs.jl. In general, inherited methods refer to vertices by indices, not labels, for compatibility with `LightGraphs.AbstractGraph`. Vertex indices get reassigned after `rem_vertex!` to remain contiguous, so we recommend using labels if possible.
```

Reply:
In this particular instance, the whole documentation and code of `MetaGraphsNext` uses the word `code` for the vertex index. I agree it is suboptimal, but it was not the subject of this PR, which simply aimed at making the documentation more accessible. I'll open an issue to keep track though

Reply:
ok that works for me

Reply:
`code` was the wording I was using. I'm not sure if `indices` is quite right because `getindex` takes labels, not codes. But if people like `indices` better we can change it.